Sources/NIO/ConvenienceOptionSupport.swift

```diff
@@ -0,0 +1,440 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftNIO open source project
+//
+// Copyright (c) 2020 Apple Inc. and the SwiftNIO project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+// MARK: ServerBootstrap - Server
+extension ServerBootstrap {
+    /// Specifies some `ChannelOption`s to be applied to the `ServerSocketChannel`.
+    /// - See: serverChannelOption
+    /// - Parameter options: List of shorthand options to apply.
+    /// - Returns: The updated server bootstrap (`self` being mutated)
+    @inlinable
+    public func serverChannelOptions(_ options: [NIOTCPServerShorthandOption]) -> ServerBootstrap {
+        var toReturn = self
+        for option in options {
+            toReturn = toReturn.serverChannelOption(option)
+        }
+        return toReturn
+    }
+    
+    @usableFromInline
+    func serverChannelOption(_ option : NIOTCPServerShorthandOption) -> ServerBootstrap {
+        let applier = ServerBootstrapServer_Applier(contained: self)
+        return option.applyOptionDefaultMapping(with: applier).contained
+    }
+    
+    fileprivate struct ServerBootstrapServer_Applier : NIOChannelOptionAppliable {
+        var contained : ServerBootstrap
```

Comment:
nit: no spaces before `:`, you could just search for ` :` and replace it by `:` in this file, should get them all

Reply:
will do.